src/__swaps__/screens/Swap/providers/swap-provider.tsx

```diff
@@ -22,7 +22,7 @@ import { NavigationSteps, useSwapNavigation } from '@/__swaps__/screens/Swap/hoo
 import { useSwapSettings } from '@/__swaps__/screens/Swap/hooks/useSwapSettings';
 import { useSwapTextStyles } from '@/__swaps__/screens/Swap/hooks/useSwapTextStyles';
 import { SwapWarningType, useSwapWarning } from '@/__swaps__/screens/Swap/hooks/useSwapWarning';
-import { userAssetsQueryKey as swapsUserAssetsQueryKey } from '@/__swaps__/screens/Swap/resources/assets/userAssets';
+import { userAssetsQueryKey as swapsUserAssetsQueryKey, userAssetsQueryKey } from '@/__swaps__/screens/Swap/resources/assets/userAssets';
```

Comment:
this is importing the same thing `userAssetsQueryKey` twice but just renaming one of them - do you just want to stick with the `userAssetsQueryKey` import?

Reply:
and then you can delete lines 313 - 319

Reply:
OH weird nice spot